providers/openlineage/docs/guides/developer.rst

```diff
@@ -405,7 +405,8 @@ Writing a custom facet function
 .. code-block:: python
 
     import attrs
-    from airflow.models.taskinstance import TaskInstance, TaskInstanceState
+    from airflow.models.taskinstance import TaskInstance
+    from airflow.sdk import TaskInstanceState
```

Comment:
We cannot do this until we bump the min version of OL provider to be Airflow 3. Remove this occurrence for now.